src/components/header/HeaderComponent.tsx

```diff
@@ -116,7 +120,14 @@ export default class HeaderComponent extends React.Component<Props, State> {
           </Left>
         )}
         <Body style={style.bodyHeader}>
-          <Title style={subTitle ? [style.titleHeader, style.titleHeaderWithSubTitle] : style.titleHeader}>{title}</Title>
+          {!carModel && (
+            <Title
+              style={[
+                subTitle ? [style.titleHeader, style.titleHeaderWithSubTitle] : style.titleHeader,
+                carModel ? [style.titleHeader, style.titleHeaderWithSubTitle] : style.titleHeader,
+                carLicensePlate ? [style.titleHeader, style.titleHeaderWithSubTitle] : style.titleHeader]}>{title}</Title>)}
```

Comment:
if you render when carModel is null/undefined, no need to check again if carModel is defined in the style